linux_os/guide/system/software/sudo/sudo_custom_logfile/rule.yml

```diff
@@ -0,0 +1,27 @@
+documentation_complete: true
+
+title: 'Ensure Sudo Logfile Exists - sudo logfile'
+
+description: |-
+    A custom logfile can be configured for sudo with the logfile tag.
+
+rationale: |-
+    A sudo log file simplifies auditing of sudo commands.
+
+severity: low
+
```

Comment:
Would you mind adding a CCE too?
Something like:
https://github.com/ComplianceAsCode/content/blob/8ba767fedfc5e79ce72c5fdd8c31d718162b4e07/linux_os/guide/system/software/sudo/sudo_add_use_pty/rule.yml#L20

Take a random number from `shared/references/cce-redhat-avail.txt`.

Reply:
Actually, the tests failed due to missing CCE.